spec/acceptance/invert_spec.rb

```diff
@@ -30,7 +30,7 @@ class { '::firewall': }
 
     it 'should contain the rules' do
       shell('iptables-save') do |r|
-        if (fact('osfamily') == 'RedHat' and fact('operatingsystemmajrelease') == '5') or (fact('osfamily') == 'SLES' and fact('operatingsystemmajrelease') == '10')
+        if (fact('osfamily') == 'RedHat' and fact('operatingsystemmajrelease') == '5') or default['platform'] =~ /sles-10/
```

Comment:
default['platform'] =~ /sles-10/ should be wrapped in round brackets, for readability ?